### PR TITLE
Fix JWT expiration default and validation

### PR DESCRIPTION
**Pull request by mattwiller**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mattwiller has signed the CLA. Thanks for the pull request!

File: tests/lib/util/config-test.js [hunk 1]
```diff
@@ -105,6 +115,28 @@ describe('Config', function() {
...
 		});
+
+		leche.withData({
+			'missing key ID': [{privateKey: TEST_PRIVATE_KEY, passphrase: TEST_PASSPHRASE}],
+			'missing private key': [{keyID: TEST_KEY_ID, passphrase: TEST_PASSPHRASE}],
+			'missing passphrase': [{keyID: TEST_KEY_ID, privateKey: TEST_PRIVATE_KEY}],
+			'unsupported algorithm': [Object.assign({algorithm: 'HS4096'}, TEST_APP_AUTH_PARAMS)],
+			'float expiration time': [Object.assign({expirationTime: 5.5}, TEST_APP_AUTH_PARAMS)],
+			'zero expiration time': [Object.assign({expirationTime: 0}, TEST_APP_AUTH_PARAMS)],
+			'too-long expiration time': [Object.assign({expirationTime: 120}, TEST_APP_AUTH_PARAMS)]
```
Comment (djordan) [inline tests/lib/util/config-test.js]: I'd recommend testing 0, 60, and 61 to stay around the boundary conditions.
